packages/bs-reform/src/ReForm.re

```diff
@@ -1,20 +1,31 @@
+open ReSchema;
+
 module type Config = {
   type field('a);
   type state;
   let set: (state, field('a), 'a) => state;
   let get: (state, field('a)) => 'a;
 };
+
 type fieldState =
   | Pristine
   | Valid
+  | NestedErrors(array(ReSchema.childFieldError))
   | Error(string);
+
+type fieldError =
```

Comment:
hmm why not add to the `fieldState` Error constructor?

Reply:
Good point. I think initially I was thinking it made sense to have it separate so users have simpler access to the error within the fieldInterface, but now I'm thinking it just adds bloat.